server/src/routes/api/v1/votesRouter.js

```diff
@@ -0,0 +1,57 @@
+import express from 'express'
+import { Vote } from '../../../models/index.js'
+import VoteSerializer from '../../../../serializers/VoteSerializer.js'
+import { Review } from '../../../models/index.js'
+
+const votesRouter = new express.Router()
+
+votesRouter.post('/initial', async (req, res) => {
```

Comment:
Following RESTful requests, this should be a GET path since we are not creating any data